src/workerd/api/queue.c++

```diff
@@ -272,6 +302,11 @@ kj::Promise<void> WorkerQueue::sendBatch(jsg::Lock& js, jsg::Sequence<MessageSen
       bodyBuilder.add('"');
     }
 
+    KJ_IF_SOME(delaySecs, serializedBodies[i].delaySecs) {
+      bodyBuilder.addAll(",\"delaySecs\": "_kj);
```

Comment:
If we're adding additional fields into the JSON, do we need to allocate a larger buffer up above? (see the `estimatedSize` calculation)